src/System.ComponentModel.Annotations/pkg/System.ComponentModel.Annotations.pkgproj

```diff
@@ -17,6 +17,7 @@
     </InboxOnTargetFramework>
     <InboxOnTargetFramework Include="netcoreapp2.0" />
     <InboxOnTargetFramework Include="$(UAPvNextTFM)" />
+    <InboxOnTargetFramework Include="uap10.0.15138" />
```

Comment:
This PR seems to cover the ones that are marked as Inbox but do we have any that actually included assets that we now need to harvest?

Reply:
Ok I have changed the approach and instead just fixed the package index for now. I will later send a subsequent PR that fixes all packages so that they are all marked as inbox for uap.